source/connect-atlas-cli.txt

```diff
@@ -27,6 +27,16 @@ When you connect to an existing |service| account from the
 To create a new |service| account or onboard an existing account that
 doesn't have any {+clusters+}, see :ref:`atlas-cli-onboarding`.
 
+.. note::
+
+   If you have configured :ref:`API keys <atlas-admin-api-access>`
+   to access |service|, you can set :ref:`environment variables 
+   <atlas-cli-env-vars>` instead of authenticating using the 
+   commands listed above.
```

Comment:
Question: Would this be better under the Use Case column in the table itself?

Reply:
I'm not sure. I think the note would be compressed width-wise in such a narrow column. I think it also applies to both use cases technically. maybe @sarahsimpers can chime in here too?

Reply:
I agree with @jwilliams-mongo, it applies to both so having it at the top is best.